### PR TITLE
added_appliance_timeconfig

### PR DESCRIPTION
**Pull request by SHANDCRUZ**
### Description
Added Appliance Configuration Timeconfig Resource

### Issues Resolved
NA

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass for go 1.11 + gofmt checks.
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [ ] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: endpoints-support.md [hunk 1]
```diff
@@ -19,6 +19,8 @@
 ## HPE OneView
 | Endpoints                                                                     | Verb     | V200 | V300 | V500 |V600 |V800 | V1000 | V1200 | V1600 | V1800 | V2000 | V2200 | V2400 | V2600 | V2800 |
 | --------------------------------------------------------------------------------------- | -------- | :------------------: | :------------------: | :------------------: | :------------------: | :------------------: | :------------------: | :------------------: | :------------------: | :------------------: | :------------------: | :------------------: | :------------------: | :------------------: | :------------------: |
```
Comment (nabhajit-ray) [inline endpoints-support.md:21]: If you have tried it is api 3000 add that column

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,13 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
+# [v6.2.0]
```
Comment (AsisBagga) [inline CHANGELOG.md:4]: please add unreleased

**Review by AsisBagga — APPROVED** [review]
just add unreleased after v6.2.0

File: CHANGELOG.md (hunk 2 above)
Comment (SHANDCRUZ) [reply CHANGELOG.md:4]: yea sure..

**Review by VenkateshRavula — APPROVED** [review]

**Pull request merged**